### PR TITLE
Capture client info

### PR DESCRIPTION
**Pull request by hiroshihorie**
LK-501

Platform differences are a bit tricky.

### CODE REVIEW

**Comment by hiroshihorie** [comment]
I've implemented JS interop for `ua-parser-js` but I've realized UserAgent isn't so reliable... 😂
why is it my Chrome reporting
`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36`

**macOS 10.15.7... ?
https://chromestatus.com/feature/5452592194781184

File: lib/src/utils.dart [hunk 1]
```diff
@@ -1,30 +1,131 @@
...
 
 // Collection of state-less static methods
 class Utils {
-  static Uri buildUri(
+  // DeviceInfoPlugin caches internally
+  static final _deviceInfoPlugin = DeviceInfoPlugin();
+
+  static Future<lk_models.ClientInfo?> _clientInfo() async {
+    switch (lkPlatform()) {
+      case PlatformType.web:
```
[the remark this answers is not in the document]
Comment (hiroshihorie) [reply lib/src/utils.dart:31]: Ok boss! 😁

**Comment by hiroshihorie** [comment]
Removed uaparser dependency and updated to use [platform_detect](https://pub.dev/packages/platform_detect) package instead which can detect OS also

**Review by davidliu — APPROVED** [review]

**Review by davidzhao — APPROVED** [review]
nice!

**Pull request merged**